### DISCUSSION TITLE
multiple tags for user and vhost ?

### DISCUSSION THREAD

**Discussion by spham** [rabbitmq/rabbitmq-server#4109] [Questions] [answered]
Hi

is it possible to make something like this ?

```
rabbitmqctl set_permissions -p vhost1 user1 'administrator'
rabbitmqctl set_permissions -p vhost2 user1 ''monitoring'
```

or something like this ?

`rabbitmqctl set_user_tags user1 [{''vhost1','administrator'},{'vhost2','monitoring'}] ?`

we have some user could access to vhost1, but on vhost2, it could monitoring only, because, the project is not the same

i try this without success 
```

sudo rabbitmqctl set_user_tags -p vhost1 user1 'administrator'
sudo rabbitmqctl set_user_tags -p vhost2 user1 'monitoring'
```

**Comment by lukebakken** [comment, marked as answer]
Please, do not open new issues or discussions. Continue the discussion here -

https://github.com/rabbitmq/rabbitmq-server/discussions/4021
